spec/javascripts/components/projects/project_card_spec.js

```diff
@@ -81,6 +94,19 @@ describe('<ProjectCard />', () => {
         });
       });
 
+      describe('.card-tag', () => {
+        it('has the tag', () => {
```

Comment:
@talyssonoc I have added a test to check the present of background and foreground.